app/src/main/java/com/work/restaurant/data/source/local/road/RoadLocalDataSourceImpl.kt

```diff
@@ -73,7 +74,8 @@ class RoadLocalDataSourceImpl : RoadLocalDataSource {
     }
 
     override fun getAddressCount(callback: RoadLocalDataCountCallback) {
-        val thread = Thread {
+
+        AppExecutors().diskIO.execute {
             val addressDatabase = AddressDatabase.getInstance(App.instance.context())
```

Comment:
```suggestion
            val addressDatabase = AddressDatabase.getInstance(App.instance.context())
```
이 객체도 RoadLocalDataSourceImpl 의 생성자에서 주입받아서 사용하는 것이 효율면에서 좋을 것 같습니다.

Reply:
56dda00 수정하였습니다.